assets/js/romo/inline.js

```diff
@@ -1,80 +1,79 @@
 var RomoInline = function(element) {
-  this.elem        = $(element);
-  this.toggleElem  = $(this.elem.data('romo-inline-toggle'));
+  this.elem        = element;
+  this.toggleElem  = Romo.f(Romo.data(this.elem, 'romo-inline-toggle'))[0];
   this.dismissElem = undefined;
 
-  this.elem.on('inline:triggerDismiss', $.proxy(this.onDismissClick, this));
-  this.elem.on('inline:triggerShow',  $.proxy(function(e) {
+  Romo.on(this.elem, 'romoInline:triggerDismiss', Romo.proxy(this.onDismissClick, this));
+  Romo.on(this.elem, 'romoInline:triggerShow',  Romo.proxy(function(e) {
     this.doShow();
   }, this));
 
-  this.elem.on('romoAjax:callStart', $.proxy(function(e, romoAjax) {
+  Romo.on(this.elem, 'romoAjax:callStart', Romo.proxy(function(e, romoAjax) {
     this.doLoadStart();
     return false;
   }, this));
-  this.elem.on('romoAjax:callSuccess', $.proxy(function(e, data, romoAjax) {
+  Romo.on(this.elem, 'romoAjax:callSuccess', Romo.proxy(function(e, data, romoAjax) {
     this.doLoadSuccess(data);
     return false;
   }, this));
-  this.elem.on('romoAjax:callError', $.proxy(function(e, xhr, romoAjax) {
+  Romo.on(this.elem, 'romoAjax:callError', Romo.proxy(function(e, xhr, romoAjax) {
     this.doLoadError(xhr);
     return false;
   }, this));
 
   this.doBindDismiss();
   this.doInit();
-  this.elem.trigger('inline:ready', [this]);
+  Romo.trigger(this.elem, 'romoInline:ready', [this]);
 }
 
 RomoInline.prototype.doInit = function() {
   // override as needed
 }
 
 RomoInline.prototype.doLoadStart = function() {
-  this.elem.html('');
-  this.elem.trigger('inline:loadStart', [this]);
+  this.elem.innerHTML = '';
```

Comment:
@jcredding we discussed switching this to `Romo.updateHtml(this.elem, '');` (both here and in dropdown, modal, and tooltip too).